src/System Application/App/File System/src/FileSystem/FileSystem.Codeunit.al

```diff
@@ -0,0 +1,252 @@
+    /// <returns>Returns true if the file exists.</returns>
+    procedure FileExists(Path: Text): Boolean
+    begin
+        exit(FileSystemImpl.FileExists(Path));
+    end;
+
+    /// <summary>
+    /// Deletes a file from the file account.
+    /// </summary>
+    /// <param name="Path">File path of the file to delete.</param>
+    [TryFunction]
+    procedure DeleteFile(Path: Text)
+    begin
+        FileSystemImpl.DeleteFile(Path);
+    end;
+
+    /// <summary>
+    /// List all directories from the given path.
+    /// </summary>
+    /// <param name="Path">Folder to list</param>
+    /// <param name="FilePaginationData">Defines the pagination data.</param>
+    /// <param name="FileAccountContent">File account content.</param>
+    [TryFunction]
+    procedure ListDirectories(Path: Text; FilePaginationData: Codeunit "File Pagination Data"; var FileAccountContent: Record "File Account Content" temporary)
+    begin
+        FileSystemImpl.ListDirectories(Path, FilePaginationData, FileAccountContent);
+    end;
+
+    /// <summary>
+    /// Creates a directory in the file account.
+    /// </summary>
+    /// <param name="Path">Path of the new Directory to create.</param>
+    [TryFunction]
+    procedure CreateDirectory(Path: Text)
+    begin
+        FileSystemImpl.CreateDirectory(Path);
+    end;
+
+    /// <summary>
+    /// Checks if a specific directory exists in the file account.
+    /// </summary>
+    /// <param name="Path">Path of the directory to check.</param>
+    /// <returns>Returns true if directory exists.</returns>
+    procedure DirectoryExists(Path: Text): Boolean
+    begin
+        exit(FileSystemImpl.DirectoryExists(Path));
+    end;
+
+    /// <summary>
+    /// Deletes a directory from the file account.
+    /// </summary>
+    /// <param name="Path">Directory to remove.</param>
+    [TryFunction]
+    procedure DeleteDirectory(Path: Text)
+    begin
+        FileSystemImpl.DeleteDirectory(Path);
+    end;
+
+    /// <summary>
+    /// Combines to paths together.
+    /// </summary>
+    /// <param name="Path">First part to combine.</param>
+    /// <param name="ChildPath">Second part to combine.</param>
+    /// <returns>Correctly combined path.</returns>
+    procedure CombinePath(Path: Text; ChildPath: Text): Text
+    begin
+        exit(FileSystemImpl.CombinePath(Path, ChildPath));
+    end;
+
+    /// <summary>
+    /// Gets the Parent Path of the given path.
+    /// </summary>
+    /// <param name="Path">File or directory path.</param>
+    /// <returns>The parent of the specified path.</returns>
+    procedure GetParentPath(Path: Text): Text
+    begin
+        exit(FileSystemImpl.GetParentPath(Path));
+    end;
+
+    /// <summary>
+    /// Opens a folder selection dialog.
+    /// </summary>
+    /// <param name="Path">Start path of the dialog.</param>
+    /// <returns>Returns the selected Folder.</returns>
+    procedure SelectFolderUI(Path: Text): Text
```

Comment:
I'm not a fan of `UI` at the end of the method name. 

Maybe remove the `UI` part, or a name like `SelectAndGetFolderPath` which is more explicit as it does 2 things. Opening a dialog to select and then returning the folder path.
Same for the other methods